py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -2278,6 +2279,7 @@ def aten_ops_bitwise_not(
 
 @dynamo_tensorrt_converter(torch.ops.aten.eq.Tensor)
 @dynamo_tensorrt_converter(torch.ops.aten.eq.Scalar)
+@dynamo_tensorrt_converter(operator.eq, supports_dynamic_shapes=True)
```

Comment:
I will rebase after https://github.com/pytorch/TensorRT/pull/2979/files#diff-9cbf43cfb62cdf682eef77f6fd5cabc488bb5a91ff09bfee5d593827542b2476R2256-R2257 is merged